::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 15 Dec 2017 19:57:46 +0100
Subject: [PATCH] glusterfs: Use 3.12 for AArch64

Currently, the glusterfs PPA for 3.13 does not have arm64 binaries,
so stick to 3.12 for now.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml            | 2 +-
 mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml    | 2 +-
 mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_control.yml.j2 | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml b/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml
index 84b392e6..1209f312 100644
--- a/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml
+++ b/mcp/reclass/classes/cluster/mcp-pike-common-ha/infra/kvm.yml
@@ -28,7 +28,7 @@ classes:
 parameters:
   _param:
     linux_system_codename: xenial
-    glusterfs_version: '3.13'
+    glusterfs_version: '3.12'
     cluster_vip_address: ${_param:infra_kvm_address}
     cluster_node01_address: ${_param:infra_kvm_node01_address}
     cluster_node02_address: ${_param:infra_kvm_node02_address}
diff --git a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml
index a6c3d9ed..d730c935 100644
--- a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml
+++ b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_compute.yml
@@ -34,7 +34,7 @@ parameters:
     keepalived_vip_interface: br-ctl
     keepalived_vip_virtual_router_id: 69
     linux_system_codename: xenial
-    glusterfs_version: '3.13'
+    glusterfs_version: '3.12'
   glusterfs:
     client:
       volumes:
diff --git a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_control.yml.j2 b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_control.yml.j2
index c487e342..62c1ab28 100644
--- a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_control.yml.j2
+++ b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_control.yml.j2
@@ -66,7 +66,7 @@ parameters:
     cluster_node03_hostname: ${_param:openstack_control_node03_hostname}
     cluster_node03_address: ${_param:openstack_control_node03_address}
     nova_vncproxy_url: https://${_param:cluster_public_host}:6080
-    glusterfs_version: '3.13'
+    glusterfs_version: '3.12'
 {%- if conf.MCP_VCP %}
   heat:
     server:
